lib/android/src/main/java/com/wix/interactable/InteractableViewManager.java

```diff
@@ -122,9 +122,9 @@ public void setAlertAreas(InteractableView view, @Nullable ReadableArray alertAr
         view.setAlertAreas(RNConvert.interactablePoints(alertAreas));
     }
 
-    @ReactProp(name = "dragWithSprings")
-    public void setDrag(InteractableView view, @Nullable ReadableMap dragWithSprings) {
-        view.setDragWithSprings(RNConvert.interactableDrag(dragWithSprings));
+    @ReactProp(name = "dragWithSpring")
```

Comment:
Ho my, did we always have `dragWithSprings` instead of `dragWithSpring`? No wonder it never worked :man_facepalming: